scripts/bot-deployment/DeployReporterCR.sh

```diff
@@ -0,0 +1,53 @@
+#!/bin/bash
+set -e
+
+if [ $# -ne 1 ]; then
+    echo "Incorrect number of arguments supplied! First and only argument is bot's name. From this the bot's config."
```

Comment:
"From this the bot's config" is there meant to be more to this sentence?

Reply:
fixed.